src/migrations/20210215181557-convert-string-types-to-text.js

```diff
@@ -1,17 +1,17 @@
 module.exports = {
-  up: async (queryInterface, Sequelize) => {
+  up: (queryInterface, Sequelize) => (
     Promise.all([
       queryInterface.changeColumn('ActivityReports', 'additionalNotes', { type: Sequelize.TEXT }),
       queryInterface.changeColumn('ActivityReports', 'context', { type: Sequelize.TEXT }),
       queryInterface.changeColumn('ActivityReports', 'resourcesUsed', { type: Sequelize.TEXT }),
-    ]);
-  },
+    ])
+  ),
 
-  down: async (queryInterface, Sequelize) => {
+  down: (queryInterface, Sequelize) => (
     Promise.all([
       queryInterface.changeColumn('ActivityReports', 'additionalNotes', { type: Sequelize.STRING }),
       queryInterface.changeColumn('ActivityReports', 'context', { type: Sequelize.STRING }),
       queryInterface.changeColumn('ActivityReports', 'resourcesUsed', { type: Sequelize.STRING }),
-    ]);
-  },
+    ])
+  ),
```

Comment:
Looks like we missed wrapping these into a transaction. Suggested change: (btw it wouldn't let me suggest over deleted lines)

`module.exports = {
  up: (queryInterface, Sequelize) => queryInterface.sequelize.transaction((t) => Promise.all([
    queryInterface.changeColumn('ActivityReports', 'additionalNotes', { type: Sequelize.TEXT }, { transaction: t }),
    queryInterface.changeColumn('ActivityReports', 'context', { type: Sequelize.TEXT }, { transaction: t }),
    queryInterface.changeColumn('ActivityReports', 'resourcesUsed', { type: Sequelize.TEXT }, { transaction: t }),
  ])),

  down: (queryInterface, Sequelize) => queryInterface.sequelize.transaction((t) => Promise.all([
    queryInterface.changeColumn('ActivityReports', 'additionalNotes', { type: Sequelize.STRING }, { transaction: t }),
    queryInterface.changeColumn('ActivityReports', 'context', { type: Sequelize.STRING }, { transaction: t }),
    queryInterface.changeColumn('ActivityReports', 'resourcesUsed', { type: Sequelize.STRING }, { transaction: t }),
  ])),
};`